### PR TITLE
add option to handover room admin

### PR DESCRIPTION
**Pull request by jamesql**
per #1097

- Adds option to change room owner or 'creator'
- Fixes typo li/translations.json -> li/translation.json

### CODE REVIEW

File: kousa/lib/kousa/room.ex [hunk 1]
```diff
@@ -165,6 +165,23 @@ defmodule Kousa.Room do
       )
     end
   end
+	
+  def change_room_creator(new_creator_id, current_room_id \\ nil) do
```
Comment (nadirabbas) [inline kousa/lib/kousa/room.ex]: - reset `package-lock.json`
- also need to change room creator in the DB, `leave_room` sets the new creator for example https://github.com/benawad/dogehouse/blob/01dd6b5c18920a3458824fae1db48a4245f3aa8e/kousa/lib/beef/mutations/rooms.ex#L115
- backend validation: 
    - check if the user who initiated the request is the room creator of the room which they're in
    - check if the user_to_be_new_creator is in the same room as the room creator

**Review by nadirabbas — CHANGES_REQUESTED** [review]

File: kousa/lib/kousa/room.ex (hunk 1 above)
Comment (jamesql) [reply kousa/lib/kousa/room.ex]: no problem

Comment (nadirabbas) [reply kousa/lib/kousa/room.ex]: @jamesql I appreciate it

**Comment by jamesql** [comment]
im going to test the new commits now give me about 10 minutes

File: kousa/lib/kousa/room.ex (hunk 1 above)
Comment (jamesql) [reply kousa/lib/kousa/room.ex]: @nadirabbas take a look at it now, let me know if anything else should be changed

**Comment by jamesql** [comment]
okay package-lock should be good now haha

**Pull request merged**